source/docs/basics-of-scripting/memory-flags-and-definitions.rst

```diff
@@ -220,7 +220,72 @@ Of course, you can do a bit more with flags than you can with definitions.
 2B. Math With Flags
 ~~~~~~~~~~~~~~~~~~~
 
+You can do more than just store raw information with flags - you can also do math! Flag math operations support addition, subtraction, multiplication, and division. The operation you specify will be performed on the value currently stored. Note that this only works for flags with numerical values - if you have stored text on a flag, you cannot use math to change the value!
 
+For example, first we'll use the ``/ex`` command to store a value in a flag:
+
+.. code::
+
+    /ex flag server DoSomeMath 10
+
+That command will store the value ``10`` in the server flag ``DoSomeMath``. Now, let's do some math!
+
+.. code::
+
+    flagmath:
+        type: world
+        events:
+            on player breaks sand:
+            - flag server DoSomeMath:+:2
+            - narrate <server.flag[DoSomeMath]>
+
+This script will add 2 to the value of the server flag ``DoSomeMath`` when a player breaks sand. Then, it will narrate the value of that flag. Why don't we try subtraction?
+
+.. code::
+
+    flagmath:
+        type: world
+        events:
+            on player breaks sand:
+            - flag server DoSomeMath:-:4
+            - narrate <server.flag[DoSomeMath]>
+
+If you guessed that this script will subtract 4 from the server flag ``DoSomeMath`` each time a player breaks sand, you're correct! You can also do multiplication and division:
+
+.. code::
+
+    flagmath:
+        type: world
+        events:
+            on player breaks sand:
+            - flag server DoSomeMath:*:10
+            - narrate <server.flag[DoSomeMath]>
+
+This script will multiply the server flag ``DoSomeMath`` by 10 each time a player breaks sand.
+
+.. code::
+
+    flagmath:
+        type: world
+        events:
+            on player breaks sand:
+            - flag server DoSomeMath:/:3
+            - narrate <server.flag[DoSomeMath]>
+
+Finally, this script will divide the server flag ``DoSomeMath`` by 3 each time a player breaks sand.
+
+There are two more options for doing math with flags. You can ``- flag server MyAdditiveFlag:++`` to add 1 to the flag and ``- flag server MySubtractiveFlag:--`` to subtract one from the flag. The addition or subtraction will occur each time the command is run. You can use these commands to easily track simple variables. For example, this script will track zombie deaths:
```

Comment:
See comment for line 223.

Reply:
Additional commentary: I don't feel that this section is very clear on how the `:++` and `:--` actions work. For example, it's not clarified that they don't take a value, and that they naturally only add/subtract 1.

Additionally, flags are already introduced as a way to store values in a more persistent manner, so there's no need to reiterate that they can track simple values. Just note that these mathematical operators can be used to directly change the value of the flag, given that the flag's value is a number.

Reply:
`++` is increment, `--` is decrement.
Addition/Subtraction are higher order math. `x + y` is basically `repeat y { x++ }`
Similar to how multiplication like `x * y` is the next order above addition  `repeat y { answer += x }`